docs/docusaurus.config.ts

```diff
@@ -0,0 +1,123 @@
+import {themes as prismThemes} from 'prism-react-renderer';
+import type {Config} from '@docusaurus/types';
+import type * as Preset from '@docusaurus/preset-classic';
+
+// This runs in Node.js - Don't use client-side code here (browser APIs, JSX...)
+
+const config: Config = {
+  title: 'Kitsune2',
+  tagline: 'A peer-to-peer communication framework designed for managing communication and state across multiple spaces.',
```

Comment:
```suggestion
  tagline: 'A peer-to-peer communication framework designed for managing communication and state.',
```

Describes Kitsune2 in terms of "spaces" which are not a concept a new reader will know anything about.

Reply:
changed in https://github.com/holochain/kitsune2/pull/336/commits/cecd6576604cae3533ca594d2be2bba714e47a83